src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -37,6 +46,10 @@
 	// cache mode
 	private CacheMode cacheMode = CacheMode.MEMORY;
 
+	public final static String COMPONENT = "component";
+	public final static String RESOURCE = "resource";
```

Comment:
both constants can be move to com.vmware.vipclient.i18n.util.ConstantsKeys class

Reply:
fixed